lua/entities/gmod_tardis_interior/modules/libraries/sh_data.lua

```diff
@@ -1,80 +1,25 @@
 -- Data
 
 if SERVER then
-    util.AddNetworkString("TARDISI-Data")
-    util.AddNetworkString("TARDISI-DataClear")
-    
     function ENT:SendData(ply)
-        if self.data then
-            net.Start("TARDISI-Data")
-                net.WriteEntity(self)
-                net.WriteBool(true)
-                net.WriteString(TARDIS.von.serialize(self.data))
-            if IsValid(ply) then
-                net.Send(ply)
-            else
-                net.Broadcast()
-            end
-        end
+        self.exterior:SendData(ply)
     end
-    
+
     hook.Add("PlayerInitialSpawn", "TARDISI-Data", function(ply)
```

Comment:
This hook likely isn't required if it's sourcing from the exterior only now

Reply:
Fixed